tests/test_io.py

```diff
@@ -1269,3 +1270,98 @@ def test_read_sql_dtype_backend() -> None:
             pd.DataFrame,
         )
         conn2.close()
+
+
+@lxml_skip
```

Comment:
You only need this around the part that reads XML, so separate just that test to another function.

Reply:
Done